programmers/3일차/두개-뽑아서-더하기.js

```diff
@@ -0,0 +1,24 @@
+function solution (numbers) {
+    let arrAllNumber = [];
+    let count = 1;
+
+    for ( let i = 0; i<numbers.length; i++ ) {
+        for ( let j = count; j<numbers.length; j++) {
+            const numberSum = numbers[i]+numbers[j];
```

Comment:
```suggestion
            numberSum = numbers[i]+numbers[j];
```
이것 역시 위에서 미리 선언 후 사용해주시는게 좋아요